docs/documentation/updating-the-kit.md

```diff
@@ -1,129 +1,44 @@
-# Updating the kit
+# Updating to the latest version
 
-**Important note**
+1. Make a backup copy of your prototype folder.
```

Comment:
I'm not sure if that would simplify things - because the user would need to (or feel they need to) refer back and forth to that list while going through the steps. It's easier for the user if each step is super-clear which folder we mean. Definitely let me know if any of the steps could be clearer on that.

Reply:
That makes sense 👍

Reply:
I've added a couple more comments to clarify which folder we're talking about each step.